pandas/core/dtypes/missing.py

```diff
@@ -52,12 +53,15 @@ def isna(obj):
 
 
 def _isna_new(obj):
+    from ..arrays import ExtensionArray
```

Comment:
pls don't do this. rather define an ABCExtensionArray. instead. This is the purpose of these, to avoid smell like this.